CMakeLists.txt

```diff
@@ -1,83 +1,130 @@
 cmake_minimum_required(VERSION 3.1)
 
-project(PanzerChasm)
+project(Chasm-Reverse)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
I think it is not a good idea to change C++ standard in such project. Some people still may want to build it with relatively-old compiler.